docs/src/components/home/ProductSuite.tsx

```diff
@@ -56,11 +56,10 @@ const ProductSuite = () => {
                 overline="Products"
                 title={
                   <Typography variant="h2" sx={{ my: 1 }}>
-                    Extensive library of components, ready for{' '}
-                    <GradientText>production</GradientText>
+                    Every component you need is <GradientText>ready to ship</GradientText> today
```

Comment:
Front end dev teams ship UIs, not components.

Reply:
Would it make more sense if we followed the original text more closely?

```suggestion
                    Every component you need is <GradientText>ready for production</GradientText> today
```

Reply:
Better